install.sh

```diff
@@ -34,6 +34,8 @@ LATEST_RELEASE=$(curl -L -s -H 'Accept: application/json' https://github.com/atu
 # Allow sed; sometimes it's more readable than ${variable//search/replace}
 # shellcheck disable=SC2001
 LATEST_VERSION=$(echo "$LATEST_RELEASE" | sed -e 's/.*"tag_name":"\([^"]*\)".*/\1/')
+# For binary install, use the first argument as the installation directory.
+INSTALL_DIRECTORY=${1:-/usr/local/bin}
```

Comment:
I'd really rather we just made this an env var input. That way we don't change the interface _at all_, but still allow customization.

Reply:
This makes sense.